Assets/Scripts/Controllers/WorldController.cs

```diff
@@ -119,15 +98,14 @@ public void Start()
         buildModeController = new BuildModeController();
         spawnInventoryController = new SpawnInventoryController();
         mouseController = new MouseController(buildModeController, furnitureSpriteController, utilitySpriteController, circleCursorPrefab);
-        keyboardManager = KeyboardManager.Instance;
         questController = new QuestController();
         cameraController = new CameraController();
         TradeController = new TradeController();
         autosaveManager = new AutosaveManager();
 
         // Register inputs actions
-        keyboardManager.RegisterInputAction("Pause", KeyboardMappedInputType.KeyUp, () => { IsPaused = !IsPaused; });
-        keyboardManager.RegisterInputAction("DevMode", KeyboardMappedInputType.KeyDown, ChangeDevMode);
+        KeyboardManager.Instance.RegisterInputAction("Pause", KeyboardMappedInputType.KeyUp, () => { GameController.Instance.IsPaused = !GameController.Instance.IsPaused; });
```

Comment:
We should probably move this line to the GameController. Makes more sense to have it there now.